src/os/bluestore/BlueStore.cc

```diff
@@ -18423,7 +18443,7 @@ int BlueStore::reconstruct_allocations(Allocator* allocator, read_alloc_stats_t
 int BlueStore::read_allocation_from_drive_on_startup()
 {
   int ret = 0;
-  dout(5) << "Start Allocation Recovery from ONodes ..." << dendl;
+  dout(0) << "Start Allocation Recovery from ONodes ..." << dendl;
```

Comment:
IMO an overkill. There are two dout(1)-s before calling this func and one more dout(0) here!!! Single one is enough I think...

Reply:
Agree.

Reply:
The douts on levels <5 should provide short, precise information.